src/gui/DatabaseOpenWidget.cpp

```diff
@@ -137,6 +137,14 @@ void DatabaseOpenWidget::load(const QString& filename)
     clearForms();
 
     m_filename = filename;
+    if (m_filename.contains(".old.")) {
+        auto* msgBox = new QMessageBox();
```

Comment:
This should either be stack-allocated or parented to `this`. I would also check explicitly for `endswith(".old.kdbx")`.